docs/source/tracking.rst

```diff
@@ -212,6 +212,13 @@ Enabling the Tracking Server to perform proxied artifact access in order to rout
   * Retrieving artifacts from the configured backend store for a user request is done with the same authorized authentication that was configured at server start
   * Artifacts are passed to the end user through the Tracking Server through the interface of the ``HttpArtifactRepository``
 
+.. note::
+    When an experiment is created, the artifact storage location from the configuration of the tracking server is logged in the experiment's metadata.
+    If enabling proxied artifact storage, any existing experiments that were created while operating a tracking server in non-proxy mode will not
+    be available for adding runs to them. This is due to the default artifact storage locations (``artifact_uri``) of the old experiments logged prior to
```

Comment:
```
If enabling proxied artifact storage, any existing experiments that were created while operating a tracking server in non-proxy mode will not
    be available for adding runs to them
```

This isn't quite accurate, right? Clients can still log to the old experiment as long as they have sufficient credentials.

Reply:
That definitely could have been worded more clearly and listing out details instead of assuming why someone would want to use the tracking server for proxy serving. I reworded this to focus on this security aspect and why creating new experiments might be the best after migrating to this config.